src/app/beer_garden/garden.py

```diff
@@ -186,29 +186,34 @@ def get_connection_defaults():
 
     # TODO: this is a temporary work-around until Brewtils is configured to provide
     # sensible defaults
+    bad_defaults = {"ssl_enabled", "ca_verify"}
     sensible_defaults = {
-        "bg_host": "somehostname",
+        "bg_host": "child_hostname",
         "bg_port": 1025,
         "ssl_enabled": False,
         "bg_url_prefix": "/",
-        "ca_cert": "none",
         "ca_verify": False,
-        "client_cert": "none",
     }
+
     # substitute the sensible default only if we're provided `None` or an empty string
-    defaults = {
-        key: (
-            defaults[key]
-            if (
-                (defaults[key] is not None and (isinstance(defaults[key], bool)))
-                or defaults[key]
-            )
-            else sensible_defaults[key]
-        )
-        for key in config_map
-    }
+    new_defaults = {}
```

Comment:
This was cleaned up from the previous version. Semantically the same, but there were some edge cases the other version would barf on.